src/runtime/object.cpp

```diff
@@ -2419,6 +2419,21 @@ extern "C" LEAN_EXPORT obj_res lean_byte_array_mk(obj_arg a) {
     return r;
 }
 
+extern "C" LEAN_EXPORT bool lean_byte_array_beq(obj_arg a, obj_arg b) {
+    if (a == b) {
+        return 1;
```

Comment:
It's probably more readable to use `true` and `false` rather than `1` and `0` when writing a function that returns `bool`.